base_layer/wallet_ffi/src/lib.rs

```diff
@@ -3085,7 +3056,7 @@ pub unsafe extern "C" fn wallet_create(
     callback_transaction_mined_unconfirmed: unsafe extern "C" fn(*mut TariCompletedTransaction, u64),
     callback_direct_send_result: unsafe extern "C" fn(c_ulonglong, bool),
     callback_store_and_forward_send_result: unsafe extern "C" fn(c_ulonglong, bool),
-    callback_transaction_cancellation: unsafe extern "C" fn(*mut TariCompletedTransaction),
+    callback_transaction_cancellation: unsafe extern "C" fn(*mut TariCompletedTransaction, u64),
```

Comment:
Should be documented in the docstring above and in `wallet.h` docstring what this callback does. It should include a description of the enum values.

Reply:
Whoops missed that, thanks @philipr-za